node/_fs/_fs_opendir.ts

```diff
@@ -43,15 +31,13 @@ export function opendir(
 
   path = getValidatedPath(path).toString();
 
-  options = typeof options === "object" ? options : {};
-  const encoding = options?.encoding ?? "utf8";
-  const bufferSize = options?.bufferSize ?? 32;
-
   let err, dir;
   try {
-    assertEncoding(encoding);
-
-    checkBufferSize(bufferSize, "options.bufferSize");
+    const { _encoding, bufferSize } = getOptions(options, {
```

Comment:
We don't need to destructure `_encoding` out if it's not being used. Same comment below.

Reply:
fixed https://github.com/denoland/deno_std/pull/2782/commits/fe69a42ce0484c8aebe626c2578d9456147ecc96